widget.go

```diff
@@ -108,6 +138,332 @@ var widgets = map[string]widgetInfo{
 				widget.NewFormItem("Text", entry)}
 		},
 	},
+	"*widget.Check": {
+		name: "CheckBox",
+		create: func() fyne.CanvasObject {
+			return widget.NewCheck("Tick it or don't", func(b bool) {})
+		},
+		edit: func(obj fyne.CanvasObject) []*widget.FormItem {
+			c := obj.(*widget.Check)
+			title := widget.NewEntry()
+			title.SetText(c.Text)
+			title.OnChanged = func(text string) {
+				c.Text = text
+				c.Refresh()
+			}
+			isChecked := widget.NewCheck("", func(b bool) { c.SetChecked(b) })
+			isChecked.SetChecked(c.Checked)
+			return []*widget.FormItem{
+				widget.NewFormItem("Title", title),
+				widget.NewFormItem("isChecked", isChecked)}
+		},
+	},
+	"*widget.RadioGroup": {
+		name: "RadioGroup",
+		create: func() fyne.CanvasObject {
+			return widget.NewRadioGroup([]string{"Option 1", "Option 2"}, func(s string) {})
+		},
+		edit: func(obj fyne.CanvasObject) []*widget.FormItem {
+			r := obj.(*widget.RadioGroup)
+			initialOption := widget.NewRadioGroup(r.Options, func(s string) { r.SetSelected(s) })
+			initialOption.SetSelected(r.Selected)
+			entry := widget.NewMultiLineEntry()
+			entry.SetText(strings.Join(r.Options, "\n"))
+			entry.OnChanged = func(text string) {
+				r.Options = strings.Split(text, "\n")
+				r.Refresh()
+				initialOption.Options = strings.Split(text, "\n")
+				initialOption.Refresh()
+			}
+			return []*widget.FormItem{
+				widget.NewFormItem("Options", entry),
+				widget.NewFormItem("Initial Option", initialOption)}
+		},
+	},
+	"*widget.Select": {
+		name: "Select",
+		create: func() fyne.CanvasObject {
+			return widget.NewSelect([]string{"Option 1", "Option 2"}, func(value string) {})
+		},
+		edit: func(obj fyne.CanvasObject) []*widget.FormItem {
+			s := obj.(*widget.Select)
+			initialOption := widget.NewSelect(s.Options, func(opt string) { s.SetSelected(opt) })
```

Comment:
This needs to support "(Select one)" which is the default - it shows as expected to start with but then is no longer available if you select an option.